metric-metadata/container.yaml

```diff
@@ -0,0 +1,113 @@
+type: k8s/container
```

Comment:
I think the filenames should contain the receiver name as a prefix or parent dir for multiple metadata.yaml components like this or they are ambiguous

Reply:
@pmcollins Is this something we can change in the script?

Reply:
I'm happy to continue helping with this, but would welcome any rotel involvement as well :-)

Reply:
https://github.com/splunk/collector-config-tools/issues/20